### PR TITLE
[onert/odc] Prepare full quantization parameter

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates quantizer to prepare full quantization parameter for CircleQuantizer and call fake quantization.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Draft: #12903
Related issue: #12904

### CODE REVIEW

**Review by jyoungyun — APPROVED** [review]
LGTM

File: runtime/onert/odc/Quantizer.cc [hunk 1]
```diff
@@ -31,13 +31,58 @@ namespace onert
 namespace odc
 {
 
+using QuantizerOptions = luci::CircleQuantizer::Options;
+using QuantizeType = onert::odc::QuantizeType;
```
Comment (ragmani) [inline runtime/onert/odc/Quantizer.cc:35]: (Optional) They can be moved into the below anonymous namespace scope.

File: runtime/onert/odc/Quantizer.cc [hunk 2]
```diff
@@ -47,16 +92,34 @@ int Quantizer::quantize(const char *in, const char *out, QuantizeType qtype)
...
-    options->param(AlgorithmParameters::Quantize_input_model_dtype, "float32");
-    options->param(AlgorithmParameters::Quantize_output_model_dtype,
-                   qtype == QuantizeType::ODC_QTYPE_WO_I16_SYM ? "wo_int16" : "wo_int8");
-    options->param(AlgorithmParameters::Quantize_granularity, "channel");
+      // quantize the graph
+      quantizer.quantize(graph);
+    }
+
+    // TODO Record minmax by minmax-embedder
+    throw std::runtime_error{"Not implemented yet"};
```
Comment (ragmani) [inline runtime/onert/odc/Quantizer.cc:115]: I have a question.
This line will cause the exception to always be thrown if `full_quantize` is `true`. Did you intended it?

**Review by ragmani — APPROVED** [review]
LGTM

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**